File: week3/constructor.cpp
#include<iostream>
using namespace std;

class Student
{
    public:
        Student()
        {
            string collegeName = "MVGR";
            int collegeCode=33;
            cout<<"CollegeName : "<<collegeName<<endl;
            cout<<"CollegeCode: "<<collegeCode<<endl;
        }
        Student(string fullName, double semPerentage)
        {
            cout<<"FullName : "<<fullName<<endl;
            cout<<"SemPercentage : "<<semPerentage<<endl;
        }
        ~Student(){}
};

int main()
{
    Student vanitha;
    string name;
    float value;
    cout<<"Enter your name : ";
    getline(cin,name);
    cout<<"Enter your sem percentage : ";
    cin>>value;
    Student vanitha54(name,value);
    return 0;
}

File: week4/Constructor.java
import java.util.Scanner;
class Student
{
  public
    Student()
    {
      String collegeName="MVGR";
      int collegeCode=33;
      System.out.print("CollegeName : "+collegeName+"\n"+"CollegeCode : "+collegeCode+"\n");
    }
    Student(String fullName,double semPercentage)
    {
      System.out.print("FullName : "+fullName+"\n"+"SemPercentage : "+semPercentage+"\n");
    }
}

public class Constructor
{
  public static void main(String[] args)
  {
    @SuppressWarnings("unused")
    Student vanitha=new Student();
    @SuppressWarnings("unused")
    String name;
    double percentage;
    Scanner input=new Scanner(System.in);
    System.out.print("Enter your sweet name : ");
    name=input.nextLine();
    System.out.print("Enter your semPercentage : ");
    percentage=input.nextDouble();
    Student vanitha01=new Student(name,percentage);
  }
}
